monitor.py

```diff
@@ -74,6 +72,17 @@
 if not os.path.exists(args.backup_path):
     os.mkdir(args.backup_path)
 
+# List of public workers to monitor. For instance, in production environment, the list will include the following workers:
```

Comment:
Technically, this is a list, so you should write: "Comma-separated list of worker ids to monitor.  Example: vm-clws-prod-worker-0,vm-clws-prod-worker-1"